packages/bentley/grouping-mapping-widget/package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "@bentley/grouping-mapping-widget",
+  "version": "0.0.15",
+  "description": "An iTwin.js 3D Viewer Widget that interfaces with the iTwin Reporting Platform.",
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "Reporting"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/bentley/grouping-mapping-widget"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/main.bundle.js",
+  "types": "lib/grouping-mapping-widget.d.ts",
+  "scripts": {
+    "build": "webpack && npm run copy-files ",
```

Comment:
why are you webpacking your pkg?

Reply:
To run the IModeljsLibraryImportsPlugin to avoid the whole issue of having multiple instances of the static variables.  It is a modified version the webpack config from the build tools since there was a lot of issues running the original.

Reply:
That plugin is only supposed to be used for Apps to use the old extension api

Reply:
I saw no other way to deal with that issue of multiple static instances. Hopefully when I switch to 3.0.0 and move them to peer dependencies it will go away.

Reply:
You are supposed to use a dev+peer dep strategy when building an itwinjs pkg. Direct deps when youre an app

Reply:
I believe I tried the peer dependency pattern but somehow it still failed to play nice. Also webpack or some form of bundler is needed to bundle everything in this widget e.g. sassLoader, svg's etc...

Reply:
Also webpack was needed to process those SVGSprites that were within some of the iTwin.Js libraries. As there was no other alternative.

Reply:
add this to a .d.ts file

```ts
declare module "*.svg?sprite" {
  const src: string;
  export default src;
}
```

Reply:
Thanks but I've decided to get rid of webpack and go with your suggestion of just letting CRA do the heavy lifting.